Signal/src/AppDelegate.m

```diff
@@ -505,6 +461,56 @@ - (void)applicationDidBecomeActive:(UIApplication *)application {
     // Always check prekeys after app launches, and sometimes check on app activation.
     [TSPreKeyManager checkPreKeysIfNecessary];
 
+    static dispatch_once_t onceToken;
+    dispatch_once(&onceToken, ^{
+        // At this point, potentially lengthy DB locking migrations could be running.
+        // Avoid blocking app launch by putting all further possible DB access in async thread.
+        [[TSAccountManager sharedInstance]
+            ifRegistered:YES
+                runAsync:^{
+                    DDLogInfo(@"%@ running post launch block for registered user: %@",
+                        self.tag,
+                        [TSAccountManager localNumber]);
+
+                    [TSSocketManager requestSocketOpen];
```

Comment:
I think you removed the wrong one. We want to requestSocketOpen every time we become active, right?